### ISSUE TITLE
About the WideResNet and DeeplabV3+ baseline accuracy

### ISSUE THREAD

**Issue by Autochthonal** [ENSTA-U2IS-AI/torch-uncertainty#119]
Hi there!

When I tried to use the WideResNet Baseline for the CIFAR dataset classification, there occurs a significant accuracy gap between the model output and the ideal level provided [here](https://paperswithcode.com/task/image-classification). 

And this problem also exists when I used the DeeplabV3+ baseline for Camvid dataset. I notice you have kindly provided your solution [here](https://github.com/ENSTA-U2IS-AI/torch-uncertainty/issues/113#issuecomment-2393346872), unfortunately, after I checked the source code of [Camvid datasetmodule](https://github.com/ENSTA-U2IS-AI/torch-uncertainty/blob/main/torch_uncertainty/datamodules/segmentation/camvid.py), I am not sure about whether the `basic_augment`parameter can help model reach the 65% mIoU.

 So if you can provide demos on WideResNet and DeeplabV3+ baseline that can reach better accuracy, it will help me alot.

Thanks * 3!

**Comment by o-laurent** [comment]
Hello @Autochthonal!

Sorry for not getting back to you sooner; I've been thinking about your questions. 

Concerning WideResNet on CIFAR, which dataset are you talking about, CIFAR-10 or CIFAR-100? In my experiments, WideResNet has been overfitting a lot on CIFAR-100 and needs data augmentations to work properly. We already have some of them implemented in the library, although not enough, and not the ones that according to your website (thanks for the link) have the best performance.

Concerning CamVid, did you run the experiment? It is probably also a matter of data augmentation and pre-training, as you said in #113 if I remember correctly. `basic_augment` helped a bit. Still, pre-training will likely be the game-changer here, given the very small size of the dataset. I can look at these issues, but I don't have much time in the following weeks, unfortunately.

Thank you, and have a great day.
